queries.sql

```diff
@@ -6,3 +6,18 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name <> 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN TRANSACTION;
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+SAVEPOINT del_savepoint;
+UPDATE animals SET weight_kg = weight_kg * -1;
+ROLLBACK TO SAVEPOINT del_savepoint;
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+COMMIT;
```

Comment:
- [ ] Good job adding this transaction and screenshots, however, you are missing a few transactions before this. Kindly find them circled in the image below and add written queries for themthem.
![Screenshot 2023-06-13 at 15 20 10 (2)](https://github.com/Bellagirl-maker/vet-clinic/assets/64774936/72962913-22c3-40b1-8ef3-dad783c9e049)